app/views/admin/dashboard/index.html.erb

```diff
@@ -1,51 +1,60 @@
 <h1>Admin Dashboard</h1>
-
-<section class="packaged_orders">
+save_and_open_page
```

Comment:
get this removed and then we are good to go

Reply:
done